__tests__/test.js

```diff
@@ -0,0 +1,45 @@
+const { expect } = require('chai');
+const {
+    createIdObjects,
+    fetchPageOfDataAndFilter,
+    fetchUserDataAndAddToCSV,
+    filterContributorByTime,
+    filterResponseForImportantEvents,
+    getOrThrow,
+    parseDatesFromArgv,
+} = require('../index');
+
+const env = Object.assign({}, process.env);
```

Comment:
Let's make this variable a bit more semantic - could you re-name it `envBeforeChanges` or something like that- to make it clearer what the intent of this code is?